scripts/devel/build_and_install.py

```diff
@@ -15,12 +15,14 @@
 # limitations under the License.
 
 import argparse
+import os
 import sys
 
-from ros_buildfarm.catkin_workspace import call_catkin_make_isolated
-from ros_buildfarm.catkin_workspace import clean_workspace
-from ros_buildfarm.catkin_workspace import ensure_workspace_exists
+from ros_buildfarm.argument import add_argument_build_tool
 from ros_buildfarm.common import Scope
+from ros_buildfarm.workspace import call_build_tool
+from ros_buildfarm.workspace import clean_workspace
+from ros_buildfarm.workspace import ensure_workspace_exists
 
 
 def main(argv=sys.argv[1:]):
```

Comment:
The `description` for this script in the command line parser suggests `catkin_make_isolated`, which isn't necessarily true anymore, right?

Reply:
(`build_and_test` as well)

Reply:
Good catch. Updated in da1dccfbc88322aa5f133afc96629a405a5762c5 (which will be squashed before the merge though).